src/Components/ScrivitoExtensions/AdvancedEnumEditor/AdvancedEnumEditor.scss

```diff
@@ -0,0 +1,106 @@
+.advanced-enum-editor {
+  overflow: hidden;
+
+  .enum_attribute {
+    margin: 0;
+    padding: 0 0 3px 0;
+    display: grid;
+    gap: 6px;
+    grid-template-columns: repeat(auto-fit, minmax(105px, 1fr));
```

Comment:
* [ ] FYI the current grid settings make some items stand out, which could be confusing to editors (are they "better" or "more important"?). See also the label length feedback.

<img width="234" height="244" alt="Image" src="https://github.com/user-attachments/assets/6560f7fc-d933-4717-b6d7-8c6054777875" />


With equal height (`grid-auto-rows: 1fr`):

<img width="234" height="282" alt="Image" src="https://github.com/user-attachments/assets/1c5a19e5-65cf-4d95-8f8b-58aacc4492be" />